cypress.config.ts

```diff
@@ -1,30 +1,35 @@
-import { defineConfig } from 'cypress'
+import { defineConfig } from "cypress";
 
 export default defineConfig({
-  videosFolder: 'cypress/videos',
+  videosFolder: "cypress/videos",
   video: false,
-  screenshotsFolder: 'cypress/screenshots',
-  fixturesFolder: 'cypress/fixtures',
-  reporter: 'mochawesome',
+  screenshotsFolder: "cypress/screenshots",
+  fixturesFolder: "cypress/fixtures",
+  reporter: "mochawesome",
+
   reporterOptions: {
-    reportDir: 'cypress/results',
+    reportDir: "cypress/results",
```

Comment:
Please can this sorta linting be a separate commit so we can more easily tell what is going on, other than linting? @gsambrotta

Reply:
Oh yes, sorry, I didn't see it. It was automatically corrected by Cypress CLI.